src/main/java/com/microsoft/sqlserver/jdbc/SQLServerException.java

```diff
@@ -72,6 +72,11 @@ final int getErrorCode() {
     static final int DRIVER_ERROR_INTERMITTENT_TLS_FAILED = 7;
     static final int ERROR_SOCKET_TIMEOUT = 8;
     static final int ERROR_QUERY_TIMEOUT = 9;
+    static final int DataClassificationInvalidVersion = 24;
```

Comment:
Is there a reason why you specified the error codes but didn't use any of them? You could use `DataClassificationInvalidVersion` with  `throw new SQLServerException(SQLServerException.getErrString("R_InvalidDataClsVersionNumber"), null);` for example.

I think there is actually no need for these error codes in JDBC driver.

Reply:
The code is ported from .NET driver as such, tokens and error codes not used here are not used in ADO driver too, but may be used in future expansion of this feature. I let them stay as such as they will be useful to track changes in future in comparison to ADO changes.

Reply:
Does 24 have a special meaning or this was just 24th error code in .Net driver? If that's the case, we shouldn't jump from 9 to 24. 
Please also keep the naming consistent with other error codes.